src/Pages/Playground/Playground.tsx

```diff
@@ -1,22 +1,44 @@
 import type { Component } from 'solid-js';
+import {createEffect, createSignal} from 'solid-js';
+import Request from './Request';
+import WS from '../../Helpers/WS';
+import { Conversation } from './Conversation';
 import { ApiSelector } from './components/ApiSelector';
 import { Authenticate } from './components/Authenticate';
-import { Conversation } from './components/Conversation';
 
-export const Playground: Component = () => (
-    <div data-testid="playground-page" class="text-base bg-dark-100 text-white">
+export const Playground: Component = () => {
+    const [reqs, setReqs] = createSignal([]);
+
+    const addToReqs = (value: string) => {
+        return setReqs([...reqs(), value]);
+    };
+
+    const resetReqs = () => {
+        return setReqs([]);
+    };
+
+    createEffect(() => {
+        WS.init((item: string) => {
+            addToReqs(item)
+        });
+    });
```

Comment:
nice and clean 🥇

Reply:
I wonder, do you think if WS subscription ever needs any cleanUP through solid js? (equivalent of onComponentUnmount/ return from useEffect)?